cypress/e2e/specialFeatures/liteSiteWeight/assertions/liteSiteWeight.js

```diff
@@ -0,0 +1,62 @@
+import interceptGetRequests from '../helpers/interceptGetRequests';
+import getTotalPageSize from '../helpers/getTotalPageSize';
+
+const MAX_PAGE_WEIGHT_KB = 100;
+
+const roundTo2Decimals = num => {
+  return parseFloat(num.toFixed(2));
+};
+
+export default ({ path, pageType }) => {
+  describe('', () => {
+    let allRequests = [];
+    let liveRequests = [];
+    before(() => {
+      Cypress.automation('remote:debugger:protocol', {
+        command: 'Network.setCacheDisabled',
+        params: { cacheDisabled: true },
+      });
+      cy.clearCookies();
+      cy.clearLocalStorage();
+      interceptGetRequests(allRequests);
+      cy.visit(`${path}`);
+    });
+
+    afterEach(() => {
+      allRequests = [];
+      liveRequests = [];
+    });
+
+    it(`Page weight for ${pageType} page should be less than ${MAX_PAGE_WEIGHT_KB}Kb`, () => {
+      let localPageWeight;
+
+      getTotalPageSize(allRequests)
+        .then(localSize => {
+          localPageWeight = roundTo2Decimals(localSize);
+        })
+        .then(() => {
+          interceptGetRequests(liveRequests);
+          cy.visit(`https://www.bbc.com${path}`);
+
+          getTotalPageSize(liveRequests).then(liveSize => {
+            const livePageWeight = roundTo2Decimals(liveSize);
+            const percentageDifference =
+              (100 * (localPageWeight - livePageWeight)) /
+              ((localPageWeight + livePageWeight) / 2);
```

Comment:
What's the / 2 for again? #ihatemaths

Reply:
If im honest im not sure, also hate maths and just copied this from stack overflow - but actually think this is overly complex now you've pointed it out and we can just do (value2 - value1) / (value1) * 100 instead!

Reply:
Ah no, maybe we do need to do this: https://www.calculatorsoup.com/calculators/algebra/percent-difference-calculator.php

Reply:
ahh yeah, i think thats what the original one is , so can leave it as is yay! the difference is this way means it doesn't matter the order of the numbers it just divides the difference between two numbers by the average of the two numbers. the other way i suggested is measuring percentage change instead of difference.